### PR TITLE
Translations for French - France

### PR DESCRIPTION
**Pull request by buggythepirate**
Translation for French - France

Added French translations as well (Confirmed by a native speaker at work, since my French is not that great).

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Related issue(s):
1.
2.
3.

### Checklist:

- [x] My code follows the code style of this project,
- [ ] My change requires a change to the documentation.

### CODE REVIEW

**Comment by AMBudnik** [comment]
Hi @buggythepirate 

thank you for sharing the French translation. I can see that you have already signed the CLA as well.

Please recheck the files as all the tests are failing. I will reload the Travis as it something returns false results but apparently, some lines are missing. Please check the tutorial at https://docs.handsontable.com/pro/1.15.1/tutorial-internationalization.html and the export/import part at the beginning.

**Comment by AMBudnik** [comment]
The PR is nearly perfect. Just the Travis needs the language to be listed alphabetically

```
  Expected value to equal:
      ["deCH", "deDE", "enUS", "frFR", "plPL"]
    Received:
      ["deCH", "deDE", "enUS", "plPL", "frFR"]
    
    Difference:
    
    - Expected
    + Received
    
     Array [
       "deCH",
       "deDE",
       "enUS",
    +  "plPL",
       "frFR",
    -  "plPL",
     ]
```

**Comment by buggythepirate** [comment]
@AMBudnik Just saw as well ;D It is fixed.

**Comment by AMBudnik** [comment]
That's a good eye! I will ask our devs to update the docs.

**Comment by buggythepirate** [comment]
@AMBudnik BTW, I don't know what is wrong with the snyk checks, as I cannot see the logs due to missing permissions. It might be because the fork I pushed from is not from the latest master.

**Comment by AMBudnik** [comment]
It looks like the example uses old MomentJS version. Please raise it to newer MomentJS version.

File: src/i18n/languages/fr-FR.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+const dictionary = {
+  languageCode: 'fr-FR',
+  [C.CONTEXTMENU_ITEMS_ROW_ABOVE]: 'Insérer une ligne en haut',
+  [C.CONTEXTMENU_ITEMS_ROW_BELOW]: 'Insérer une ligne en bas',
+  [C.CONTEXTMENU_ITEMS_INSERT_LEFT]: 'Insérer une colonne à gauche',
+  [C.CONTEXTMENU_ITEMS_INSERT_RIGHT]: 'Insérer une colonne à droite',
+  [C.CONTEXTMENU_ITEMS_REMOVE_ROW]: ['Supprimer une ligne', 'Supprimer les lignes'],
+  [C.CONTEXTMENU_ITEMS_REMOVE_COLUMN]: ['Supprimer une colonne', 'Supprimer les colonnes'],
+  [C.CONTEXTMENU_ITEMS_UNDO]: 'Annuler',
+  [C.CONTEXTMENU_ITEMS_REDO]: 'Répeter',
```
Comment (niio972) [inline src/i18n/languages/fr-FR.js]: A better translation of redo is 'Rétablir'

Comment (AMBudnik) [reply src/i18n/languages/fr-FR.js]: Hi @buggythepirate can I count on your feedback?

File: src/i18n/languages/fr-FR.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  [C.FILTERS_CONDITIONS_NONE]: 'Aucun',
+  [C.FILTERS_CONDITIONS_EMPTY]: 'Est vide',
+  [C.FILTERS_CONDITIONS_NOT_EMPTY]: "N'est pas vide",
+  [C.FILTERS_CONDITIONS_EQUAL]: 'Egal à',
+  [C.FILTERS_CONDITIONS_NOT_EQUAL]: 'Est différent de',
+  [C.FILTERS_CONDITIONS_BEGINS_WITH]: 'Commence par',
+  [C.FILTERS_CONDITIONS_ENDS_WITH]: 'Finit par',
+  [C.FILTERS_CONDITIONS_CONTAINS]: 'Contient',
+  [C.FILTERS_CONDITIONS_NOT_CONTAIN]: 'Ne contient pas',
+  [C.FILTERS_CONDITIONS_GREATER_THAN]: 'Plus grand que',
```
Comment (behamon) [inline src/i18n/languages/fr-FR.js]: Suggestion for change: "Supérieur à"

File: src/i18n/languages/fr-FR.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  [C.FILTERS_CONDITIONS_GREATER_THAN_OR_EQUAL]: 'Plus grand ou égal à',
```
Comment (behamon) [inline src/i18n/languages/fr-FR.js]: Suggestion for change: "Supérieur ou égal à"

File: src/i18n/languages/fr-FR.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  [C.FILTERS_CONDITIONS_LESS_THAN]: 'Plus petit que',
```
Comment (behamon) [inline src/i18n/languages/fr-FR.js]: Suggestion for change: "Inférieur à"

File: src/i18n/languages/fr-FR.js [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+  [C.FILTERS_CONDITIONS_LESS_THAN_OR_EQUAL]: 'Plus petit ou égal à',
```
Comment (behamon) [inline src/i18n/languages/fr-FR.js]: Suggestion for change: "Inférieur ou égal à"

**Comment by behamon** [comment]
Great work, would love to see that in the next update!

**Comment by AMBudnik** [comment]
Thank you for your feedback @behamon

**Comment by AMBudnik** [comment]
Hi @buggythepirate 

thank you for reviewing the changes. Can you also update the dependencies? They break the snyk tests

**Comment by buggythepirate** [comment]
@behamon @niio972 Thanks for your feedback. I updated the translation accordingly. 

@AMBudnik I'll fix the synk part later today

**Comment by buggythepirate** [comment]
@AMBudnik synk part is gone, but Travis fails and to be honest I have no clue as to why and how it is related to the translation script

**Comment by AMBudnik** [comment]
@buggythepirate I have restarted the tests and now all pass :)

**Comment by buggythepirate** [comment]
@AMBudnik Thanks :)

**Comment by AMBudnik** [comment]
Sorry for the fall start @buggythepirate 

It looks like one of the options takes too much space in the contextMenu and we cannot merge it yet. 
We will check what is the best option to handle longer titles, as I understand that the following translation

`[C.CONTEXTMENU_ITEMS_NESTED_ROWS_DETACH_CHILD]: 'Détacher de la ligne précédente'`

is the most accurate?

**Comment by buggythepirate** [comment]
@AMBudnik What is the max. number of characters you are allowing? Maybe we can work around it for the time being.

**Comment by buggythepirate** [comment]
@AMBudnik 
We could change
      [C.CONTEXTMENU_ITEMS_NESTED_ROWS_INSERT_CHILD]: 'Insérer une sous-ligne',
      [C.CONTEXTMENU_ITEMS_NESTED_ROWS_DETACH_CHILD]: 'Détacher de la ligne précédente',

to
      [C.CONTEXTMENU_ITEMS_NESTED_ROWS_INSERT_CHILD]: 'Insérer une ligne enfant',
      [C.CONTEXTMENU_ITEMS_NESTED_ROWS_DETACH_CHILD]: 'Détacher la ligne enfant',

That could work. It sounds a bit technical (e.g. like sg. from an SQL documentation): "insert child (row)", "detach child (row)". 

@niio972 @behamon Maybe the native speakers can weigh in?

**Comment by AMBudnik** [comment]
Sorry for the delay. I have asked our programmer if the second option fits better

**Comment by AMBudnik** [comment]
Can I ask you to create a new PR as this one has been officially merged?

ps. the `Détacher la ligne enfant` works well. It fits inside the menu

**Pull request merged**